### DISCUSSION TITLE
Fatal Error with TweakScale Firespitter

### DISCUSSION THREAD

**Discussion by Ikof07** [TweakScale/TweakScale#337] [Support] [answered]
I tried installing firespitter for ksp because im trying to download all the mods for the community tech tree, and when I opened ksp, there was a fatal error. I have no clue how to fix it as I'm relatively new to modding and horrible at manually installing mods, so if its something silly then id like to apologize. I also tried attaching pics of my mods and the error message. Is there any way this can be fixed?
![image_2024-09-07_144621634](https://github.com/user-attachments/assets/6c44c936-b4a4-4a03-8411-9eee99af674f)
![image](https://github.com/user-attachments/assets/14cd117b-fd07-430f-97ee-eabcc86c8da4)

**Comment by Lisias** [comment]
Hi, @Ikof07 

I need your `KSP.log`, as explained in the message, in order to check who is the one borking on you.

Look for a file called `KSP.log` on the same place you will find the `KSP_x64.exe` file, and attach it on a post on this issue.

**Reply by Ikof07** [reply]
Apologies, here's the logs. I found what looks like an error before the fatal error as well if that helps.
![image](https://github.com/user-attachments/assets/3002bf37-3dd1-4130-a757-7eff35d57229)
![image](https://github.com/user-attachments/assets/d47b8a97-b549-416b-9935-d84563acb8bd)

**Reply by Lisias** [reply]
Sorry, dude. I need the `KSP.log` file, not a screenshot of it.

Please drag and drop the file. I need it a whole!

**Reply by Ikof07** [reply]
Ohhhh my bad. Here it is 😅
[KSP.log](https://github.com/user-attachments/files/16923911/KSP.log)

**Reply by Lisias** [reply]
Life got "interesting" for me in the last days. Sorry the delay.

Unfortunately, your `KSP.log` came truncated. I forgot to advise (my fault, sorry) to reproduce the problem **and then** quit KSP to make sure the `KSP.log` will be intact before sending it.

Sorry.

Please try again - this time, keep KSP running until the problem is reproduced and then quit KSP before copying the `KSP.log`.

**Reply by Ikof07** [reply, marked as answer]
It's fine! I ended up deleting a ton of mods that i didn't really need, including Firespitter, and the game ran fine. If I ever any similar problems, I will let you know. Sorry for the hassle and I hope you're doing well! :]
